packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -193,6 +204,13 @@ export function getPropGetters<
         // See explanation in `onTouchStart`.
         if (!isTouchDevice) {
           store.dispatch('blur', null);
+
+          // If requests are still running when the user closes the panel, they
+          // could reopen the panel once they resolve.
+          // We want to avoid any subsequent query and keep the panel closed.
```

Comment:
```suggestion
          // If requests are still running when the user closes the panel, they
          // could reopen the panel once they resolve.
          // We want to prevent any subsequent query from reopening the panel because
          // it would result in an unsolicited UI behavior.
```

Reply:
Addressed in 8280fb6.